### DISCUSSION TITLE
怎么今天早上打开设置界面进不去一直转圈呢？

### DISCUSSION THREAD

**Discussion by yangnis** [shuax/MouseInc.Settings#337] [Q&A]
怎么今天早上打开设置界面进不去一直转圈呢？哪位大佬知道请教一下

**Comment by shuax** [comment]
需要主程序更新

**Reply by yangnis** [reply]
大佬更新快出了吗？因为我是前天下载的，点那个检查更新也显示错误

**Reply by eternity49** [reply]
手动更新后解决问题

**Comment by shuax** [comment]
https://dl.shuax.com/MouseInc2.13.0.7z
手动下更新吧

**Reply by yangnis** [reply]
谢谢

**Reply by yinlsea** [reply]
下载不了

**Reply by ironqing** [reply]
shuax大佬，能否建个MouceInc软件交友群啊，我感觉很有必要，这个鼠标手势软件用户挺多的，我也一直在用，非常喜欢好用

**Reply by justinzhuying** [reply]
能不能开放OCR的接口，我们自定义自己的

**Reply by xsb0624** [reply]
> shuax大佬，能否建个MouceInc软件交友群啊，我感觉很有必要，这个鼠标手势软件用户挺多的，我也一直在用，非常喜欢好用

建群需要有人管理的，还不如就用github，或者语雀

**Reply by CLinhai** [reply]
谢谢 有效

**Reply by SoyKoa** [reply]
以前有qq群的，，后来解散了

**Comment by Robertxjf** [comment]
建议把版本号大于[2.13](https://dl.shuax.com/MouseInc2.13.0.7z) **写进本项目的readme中.**.. 找了半天；
这算是重要依赖了吧
